apps/ui/src/apis/communities/fetchAllCommunities.ts

```diff
@@ -42,19 +42,21 @@ const fetchAllCommunities = async (): Promise<Community[]> => {
         }
       }
     }`;
-  
+
   const data = await graphqlRequest<CommunitiesData>(query);
 
   return data.allCommunities.nodes.map((node): Community => {
-    const pictures = node.userCommunitiesByCommunityId.nodes.map(userCommunity => userCommunity.userByUserId.profileImage).filter((url): url is string => url != null);
+    const pictures = node.userCommunitiesByCommunityId.nodes
+      .map((userCommunity) => userCommunity.userByUserId.profileImage)
+      .filter((url): url is string => url != null);
     return {
```

Comment:
this can be changed to `{...node, picturesUrl: pictures, numberOfMembers: node.bla.bla.length}`